integration/spark/src/main/scala/org/apache/spark/sql/CarbonEnv.scala

```diff
@@ -74,7 +75,13 @@ class CarbonEnv {
 
     sparkSession.udf.register("getTupleId", () => "")
     sparkSession.udf.register("getPositionId", () => "")
-    sparkSession.udf.register("NI", (anyRef: AnyRef) => true)
+    CreateFunctionCommand(None,
```

Comment:
move None to next line and give parameter name for all parameters

Reply:
done